File: player-lib/src/main/java/com/gtbluesky/mediamatrix/player/util/TimeUtil.java
package com.gtbluesky.mediamatrix.player.util;

import java.text.SimpleDateFormat;
import java.util.TimeZone;

public class TimeUtil {

    public static String getFormatTime(int time) {
        return getFormatTime(time, "mm:ss");
    }

    public static String getFormatTime(int time, String pattern) {
        try {
            SimpleDateFormat formatter = new SimpleDateFormat(pattern);
            formatter.setTimeZone(TimeZone.getTimeZone("GMT+00:00"));
            return formatter.format(time);
        } catch (Exception e) {
            LogUtil.e(e.getMessage());
            return "00:00";
        }
    }

}

File: player-lib/src/main/java/com/gtbluesky/mediamatrix/player/util/TimeUtil.kt
package com.gtbluesky.mediamatrix.player.util

import com.gtbluesky.mediamatrix.player.util.LogUtil.e
import java.text.SimpleDateFormat
import java.util.*

object TimeUtil {
    fun getFormatTime(time: Int): String {
        return getFormatTime(time, "mm:ss")
    }

    fun getFormatTime(time: Int, pattern: String?): String {
        return try {
            val formatter = SimpleDateFormat(pattern)
            formatter.timeZone = TimeZone.getTimeZone("GMT+00:00")
            formatter.format(time)
        } catch (e: Exception) {
            e(e.message)
            "00:00"
        }
    }
}